src/modules/Categories/CategoryList.tsx

```diff
@@ -100,27 +103,28 @@ class CategoryList extends React.Component<Props, State> {
         </S.Header>
         <ScrollView>
           <S.View style={{ width: screen_width, height: 200 }}>
-              <S.Image 
-                  style={{height: 150}} 
-                  source={require('../../../assets/images/hungrylogo.png')} 
+              <S.Image
+                  style={{height: 150}}
+                  source={require('../../../assets/images/hungrylogo.png')}
               />
           </S.View>
           <rne.SearchBar
+            clearIcon={{color: 'black'}}
             lightTheme
             round
-            onChangeText={this.someMethod}
-            onClearText={this.someMethod}
+            onChangeText={(text) => this.searchCategory(text)}
+            onClearText={() => this.searchCategory('')}
```

Comment:
onChangeText={this.searchCategory}
// de esta manera es un poco mejor en temas de performance 

Si el onClearText retorna un null podria ser 
onClearText={this.searchCategory}

sino esta bien dejarlo asi
onClearText={() => this.searchCategory('')}